packages/backend-common/src/reading/GithubUrlReader.ts

```diff
@@ -18,7 +18,7 @@ import {
   GitHubIntegrationConfig,
   readGitHubIntegrationConfigs,
   getGitHubFileFetchUrl,
-  getGitHubRequestOptions,
+  GithubCredentialsProvider,
```

Comment:
I know we use camelCase, but the other vars in here are all GitHub.  Wondering if we want to maintain that for consistency? (vendor calls it capital H hub also)

```
getGitHubRequestOptions
      ^
GithubCredentialsProvider
   ^
```

There may be others that are preexisting, but just noticed it is all!

Reply:
We actually recently talked about it and so far it landed on: always use GitHub in proper text, but use `Github` in code identifiers basically just because it's a lot easier to read. No super strong opinions, and there is for sure a wild mixture in the code base at this point...

Reply:
(We should try to be consistent of course :))